src/Traits/MagicPropertyTrait.php

```diff
@@ -19,6 +19,32 @@
 /**
  * Provides the magic methods that allow read access
  * to magic properties.
+ *
+ * @property-read int year
```

Comment:
I agree that the magic property annotations make sense to have on the traits that provide them. However, it would be better if the added properties used the same conventions used in the rest of the library.